radixconfig.yaml

```diff
@@ -1,15 +1,19 @@
+#radixconfig production
 apiVersion: radix.equinor.com/v1
 kind: RadixApplication
 metadata:
-  name: oneseismictest
+  name: oneseismic
 spec:
   environments:
-    - name: dev
+    # stable production branch users are supposed to use
+    - name: prod
       build:
         from: radix
+    # environment that can be used for performance testing
+    # environment must be stopped when not needed
```

Comment:
You commented on this, but could it be possible to set the number of replicas to zero to avoid the environment from starting and then [scale the environment up via the CLI](https://radix.equinor.com/docs/topic-radix-cli/#examples) if needed? 

I have not used the Radix CLI yet, but it seems to be quite powerful.